### ISSUE TITLE
Less variables are not processed on rebuilds

### ISSUE THREAD

**Issue by iloveip** [alienCY/gatsby-antd-starter#1]
Hi there,

Thank you very much for your starter. I have a question about less variables in this file https://github.com/alienCY/gatsby-antd-starter/blob/master/src/theme/vars.less. I've noticed that they are processed only on initial build. But if I change the file during the development, while the local server is running, nothing changes. Is there any way to pick up the changes in this file on rebuilds as well?

**Comment by alienCY** [comment]
Yes sadly you have to restart the server for changes to upload. Hot reloading is not enabled atm. I was planning to change it at some point if anybody noticed my starter :3 Thank you for noticing it :) You can try writing directly the changes to Gatsby-config that way they are hot reloaded ( I think :/ ). I shall look into it later and get back to you :D

**Comment by iloveip** [comment]
Ok, thank you very much! Yes, I'm using it in my project, so any updates are highly appreciated)

**Comment by alienCY** [comment]
Hello. I know it's been a good while. Sorry for the delay. I had looked into it for a week or so and then I gave up. There is not an easy (or any) way to do it at the moment. I will try again in the future, I'm working on other stuff too and I don't really have much time for this now :/

**Comment by iloveip** [comment]
Hi @alienCY,
No problem. In the meantime I change the styles with Emotion, so I use `vars.less` just for a few variables which I don't need to update on the fly.

**Comment by devtrillo** [comment]
@alienCY
I can create a PR that fixes that

**Comment by alienCY** [comment]
@devtrillo Sure thing! That would be greatly appreciated! :D Does it work for all variables or only colors?

**Comment by devtrillo** [comment]
I use it for both, ant variables and system variables

**Comment by alienCY** [comment]
So can you make it so that changes in the .less file are hot reloaded? In any case please try to not make unnecessary changes and I am fine with merging :)
